install-deps.sh

```diff
@@ -60,7 +60,7 @@ if test -f /etc/redhat-release ; then
     $SUDO yum install -y redhat-lsb-core
 fi
 
-if type apt-get > /dev/null 2>&1 ; then
+if type apt-get > /dev/null 2>&1 && [ ! type zypper > /dev/null 2>&1 ]; then
```

Comment:
@jan--f Shouldn't this line be:

```
if type apt-get > /dev/null 2>&1 && ! type zypper > /dev/null 2>&1 ; then
```

(I tested it and the square brackets cause the conditional to behave differently than expected.)